README.md

```diff
@@ -9,44 +9,210 @@ Welcome to Traffic Courts Online
 ## Project Structure
 
 ```
-    ├── .github                                 # Contains GitHub Related sources
-    ├── docs                                    # docs and images
-    │   └── data-model                          # Oracle designer data model
+    ├── .docker
+    │   └── keycloak-realms
+    ├── .github
+    │   └── workflows
+    ├── .gitops
+    │   ├── azure
+    │   └── charts
+    │       ├── jaeger-aio
+    │       └── traffic-court-online
+    ├── docs
+    ├── infrastructure
+    │   ├── certificates
+    │   ├── keycloak
+    │   └── openshift
     ├── splunk-dash-board
-    ├── src                                      # application source files
-    │   ├── backend                              # Backend code
-    │       └── oracle-data-interface            # An Oracle Interface API
-    │   └── frontend                             # Frontend code
-    ├── tools
+    ├── src
+    │   ├── backend
+    │   │   ├── TrafficCourts
```

Comment:
Maybe we can clean up the unused src\backend\TrafficCourtsApi at some point.

Reply:
right, good idea. I wanted to limit this change to the README.md file for now.